src/Exceptionless.Web/ClientApp/app.config.js

```diff
@@ -0,0 +1,19 @@
+(function () {
+  'use strict';
+
+  angular.module('app.config', [])
+    .constant('BASE_URL', '')
```

Comment:
running locally or against prod, this is no longer set and causes web sockets and project configure page not to work as expected.

Reply:
Updated